AnomalyDetection.py

```diff
@@ -25,28 +25,33 @@ def _partition_anomalies(windows, k):
     """
     diffs = [windows[iw - 1][1] - windows[iw][1]
              for iw in range(1, len(windows))]
-    top_jump_positions = sorted(range(len(diffs)), key=lambda i: diffs[i], reverse=True)[0:k]
-    return sorted(top_jump_positions)
+    top_jump_positions = sorted(range(len(diffs)), key=lambda i: diffs[i], reverse=True)[0:k-1]
+    return sorted(top_jump_positions) + [len(windows) - 1]
 
```

Comment:
nit: to pick k largest jumps, priority queue would be more efficient than sorting the array.
at this stage, it probably does not matter, so it is good as it is.